lib/logstash/filters/fingerprint.rb

```diff
@@ -146,4 +151,16 @@ def select_digest(method)
       raise(LogStash::ConfigurationError, "Unknown digest for method=#{method.to_s}")
     end
   end
+  
+  def x64?
+    @x64 ||= begin
+      if 1.size == 8
```

Comment:
I'm confused. This looks like a platform architecture check to see the storage size of an integer. Why?